### PR TITLE
Basic support for processing endpoints

### PR DESCRIPTION
**Pull request by croudet**
- User defined endpoints
- Built-in micronaut endpoints
- Sensitiveness not supported
- Using different port for endpoints not supported.

### CODE REVIEW

**Comment by graemerocher** [comment]
Wow! Well done.. this is awesome 👍

Will review soon.

File: gradle.properties [hunk 1]
```diff
@@ -1,7 +1,7 @@
 projectVersion=1.4.5.BUILD-SNAPSHOT
```
Comment (graemerocher) [inline gradle.properties:1]: Can you bump the version to `2.0.0.BUILD-SNAPSHOT` since this changes the minimum version required to be Micronaut 1.3.4

Comment (croudet) [reply gradle.properties:1]: Oh, it works with 1.1.4. I just tested it with 1.3.4 and forgot to put it back to its initial value.

**Comment by graemerocher** [comment]
Sorry @croudet could you resolve the merge conflict? Tried to but don't think I got it right

**Comment by croudet** [comment]
I have rebased the branch. Everything looks ok.

**Comment by graemerocher** [comment]
Thanks!

**Pull request merged**